specificatie/BRK-Bevragen/unresolved/openapi.yaml

```diff
@@ -16,17 +16,31 @@ paths:
   /kadastraalonroerendezaken:
     get:
       operationId: GetKadastraalOnroerendeZaken
-      description: "Het ophalen van een verzameling Kadastraal Onroerende Zaken. "
+      description: |
+          Het ophalen van een collectie Kadastraal Onroerende zaken. 
+          Ten minste één van de volgende combinaties van parameters moet zijn opgenomen 
+          1.  Kadastrale aanduiding
+              -  kadastrale_gemeente (verplicht)
+              -  perceelnummer (verplicht)
+              -  sectie (verplicht)
+              -  appartementsrechtnummer (optioneel)
+          2.  Ingeschreven persoon als zakelijk gerechtigde
+              -  burgerservicenummer
+          3.  Niet ingeschreven persoon of niet natuurlijk persoon als zakelijk gerechtigde 
+              -  kadastraalpersoonidentificatie
+          4.  Type zakelijk gerechtigden
+              -  typegerechtigde
```

Comment:
betekent dit dat iemand kadastraalonroerendezaken kan zoeken met als enige queryparameter typegerechtigde? Dat levert toch enorm veel resultaten op? Als iemand zoekt met typegerechtigde=2 dan krijg je denk ik elke koopwoning in Nederland terug. Ik zou denken dat je typegerechtigde altijd/alleen in combinatie met de identificatie van de zakelijk gerechtigde gebruikt.

Reply:
Klopt inderdaad. Moet altijd in combinatie met identificatie van zakelijk gerechtigde.

Reply:
Aangepast